core/tests/upgrade-test/tests/upgrade.test.ts

```diff
@@ -265,12 +266,18 @@ describe('Upgrade test', function () {
             'contracts/system-contracts/artifacts-zk/contracts-preprocessed/DefaultAccount.sol/DefaultAccount.json'
         );
 
+        const evmEmulatorCode = readCode(
+            'contracts/system-contracts/zkout/EvmEmulator.yul/contracts-preprocessed/EvmEmulator.yul.json'
```

Comment:
Output path from compiler was changed several times recently, so we have some ugly functions to read bytecode. Iiuc, it isn't guaranteed that this hardcoded path will remain valid in the future (but I hope so)